### ISSUE TITLE
Invalid Version: undefined when installing on ember-cli 2.6.0

### ISSUE THREAD

**Issue by breyno127** [ember-animation/liquid-fire#503]
I recently tried to `ember install liquid-fire` on a project using `ember-cli: 2.6.0` and got the following error. Is this related to either #452 or #496, a separate issue all-together or some sort of conflict within my project?

```
$ ember install liquid-fire
Invalid Version: undefined
TypeError: Invalid Version: undefined
at new SemVer (/Users/me/my-project/node_modules/liquid-fire/node_modules/ember-cli-version-checker/node_modules/semver/semver.js:281:11)
at compare (/Users/me/my-project/node_modules/liquid-fire/node_modules/ember-cli-version-checker/node_modules/semver/semver.js:571:10)
at Function.gt (/Users/me/my-project/node_modules/liquid-fire/node_modules/ember-cli-version-checker/node_modules/semver/semver.js:600:10)
at DependencyVersionChecker.isAbove (/Users/me/my-project/node_modules/liquid-fire/node_modules/ember-cli-version-checker/index.js:62:17)
at DependencyVersionChecker.assertAbove (/Users/me/my-project/node_modules/liquid-fire/node_modules/ember-cli-version-checker/index.js:80:13)
at CoreObject.module.exports.init (/Users/me/my-project/node_modules/liquid-fire/index.js:19:56)
at CoreObject.superWrapper [as init] (/Users/me/my-project/node_modules/ember-cli/node_modules/core-object/lib/assign-properties.js:32:18)
at CoreObject.Class (/Users/me/my-project/node_modules/ember-cli/node_modules/core-object/core-object.js:33:38)
at /Users/me/my-project/node_modules/ember-cli/lib/models/addons-factory.js:48:21
at visit (/Users/me/my-project/node_modules/ember-cli/lib/utilities/DAG.js:23:3)
```

**Comment by acorncom** [comment]
Confirmed.  I'm seeing the same stack track when using `ember-cli@2.8.0`, `node@4.x` and `npm@2.x`

**Comment by netes** [comment]
Same here with `ember-cli@2.8.0`.

**Comment by ef4** [comment]
Please share the output of `npm ls`.

This exception looks like it's happening while trying to check whether liquid-fire has a compatible version of velocity-animate present.

**Comment by ef4** [comment]
You may want to try master as of https://github.com/ember-animation/liquid-fire/commit/218a791ea7f311e38bdd6a8211d488d40baf82ca and see if it fixes the problem.

**Comment by breyno127** [comment]
I still am seeing the issue. Here is the output of `npm ls`

[output.txt](https://github.com/ember-animation/liquid-fire/files/478781/output.txt)

**Comment by ef4** [comment]
I think I reproduced the original error and it was indeed fixed by https://github.com/ember-animation/liquid-fire/commit/218a791ea7f311e38bdd6a8211d488d40baf82ca, which is released in 0.26.1. Please reopen here if you can still make this occur on that version.
